### PR TITLE
added unicode switch as you type.

### PR DESCRIPTION
**Pull request by bksubhuti**
added unicode as you type.. tested for search and dictionary

### CODE REVIEW

File: _dprhtml/features/search/tab.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
           </div>
         </div>
         <input type="text" id="isearch" class="form-control" placeholder="Search term"
-          data-bind="textInput: searchString" onkeydown="if(event.keyCode==13) DPRSend.sendSearch(); event.keyCode=0;" />
+            data-bind="textInput: searchString"; onkeyup="TextToUniSearch()"; onkeydown="if(event.keyCode==13) searchHandler(); event.keyCode=0;" />
```
Comment (parthopdas) [inline _dprhtml/features/search/tab.html:24]: why the change from ```DPRSend.sendSearch()``` to ```searchHandler()```?

**Comment by bksubhuti** [comment]
Probabably leftover from a line that I cut and paste from the merge.
I had an out of date for.

Sorry.

Bhante Bhikkhu Subhūti

Website:
https://AmericanMonk.org

**Comment by parthopdas** [comment]
> Probabably leftover from a line that I cut and paste from the merge. I had an out of date for. Sorry. Bhante Bhikkhu Subhūti Website: https://AmericanMonk.org

you'll need to update it and push the change again so i can approve the PR.

**Comment by bksubhuti** [comment]
I did another pull request.. need help merging master.  I copied from the github html interface to get the section needed.  probably should have copyied the whole page.. but I just did the section.

**Comment by bksubhuti** [comment]
I did update and push and did a PR

Bhante Subhuti

https://AmericanMonk.org
https://withmetta.net
